src/main/resources/application.yml

```
@@ -12,4 +12,19 @@ spring:
          fail-on-unknown-properties: true
       serialization:
          write-empty-json-arrays: false
-         fail-on-empty-beans: false
\ No newline at end of file
+         fail-on-empty-beans: false
+
+no.nav.security.jwt:
+   issuer:
+      aad:
+         discoveryurl: https://login.microsoftonline.com/${AZURE_APP_TENANT_ID}/v2.0/.well-known/openid-configuration
+         accepted_audience: ${AZURE_APP_CLIENT_ID}, api://${AZURE_APP_CLIENT_ID}
+
+management:
+   endpoints:
+      web:
+         exposure:
+            include: '*'
+   endpoint:
+      health:
+         show-details: always
```

Tenker det kan være greit å bruke det samme som i ef-sak her, burde ikke være `*` ekspomerer `env ` som vi ikke ønsler
```suggestion
management:
  endpoint.health.show-details: always
  endpoints.web:
    exposure.include: info, health, metrics, prometheus
    base-path: "/internal"
    path-mapping:
      info: "status/isAlive"
  metrics.export.prometheus.enabled: true
  metrics.web.server.auto-time-requests: true
```

---

Tok denne i egen PR @throndi , så kan merges inn fra master :)